generic/proof-useropts.el

```diff
@@ -255,7 +255,12 @@ files which are out of date with respect to the loaded buffers!"
 
 (defcustom proof-prog-name-ask
   nil
```

Comment:
Should we set it to t by default? If you think yes please change the PR.

Reply:
That would be an unexpected change in behavior for users, yes?

Reply:
Like most security fix :)

Reply:
It's been nil since the 90s, time for a change :v

Reply:
Well, won't it be kind of annoying to be prompted for the program name each time you run PG?

Reply:
In my experience it's OK. It's pre-filled with the default so you just hit RET and go unless you want some other value (in which case there's no way around it or you want better defaults, the later being easier to provide through dir-locals with this change).

Reply:
Isn't it possible to have a configurable set of allowed values, like for File Variables?

Reply:
I removed the proof-prog-name-ask commit from this PR. Continue in #208.